minecraft_discord_bot.tf

```diff
@@ -0,0 +1,9 @@
+module "minecraft_discord_bot" {
+  source = "./modules/minecraft-discord-bot"
+  config_bucket_key_name = "config.json"
+  config_bucket_name = "minecraft-discord-bot"
+  ecs_subnet_id = aws_subnet.minecraft_server_public_subnet.id
+  ecs_task_image = "622452799301.dkr.ecr.us-west-2.amazonaws.com/minecraft-discord-bot:latest"
```

Comment:
> is this public, or are we going to have to setup trust between the accounts?

That particular image link is hosted from my AWS account and is **not** public. I was thinking we would create the repo/image in your account to not have to worry about trust between the accounts. 

> Can we use a tag instead of "latest" (eventually, doesn't have to be now)?

yup, we can even split out the image and tag if we really wanted.